packages/augur-core/source/contracts/reporting/RepPriceOracle.sol

```diff
@@ -30,6 +30,7 @@ contract RepPriceOracle is IRepPriceOracle, Initializable {
         augur = _augur;
         cash = _augur.lookup("Cash");
         uniswapFactory = IUniswapV2Factory(_augur.lookup("UniswapV2Factory"));
+        endInitialization();
```

Comment:
This seems to usually be done at the top usually, probably to avoid possibility of re-enter ?